viewer/packages/cad-geometry-loaders/src/sector/SectorDownloadScheduler.test.ts

```diff
@@ -193,9 +193,109 @@ describe(SectorDownloadScheduler.name, () => {
       }
     });
   });
+
+  test('Discarded sectors should be aborted if still pending', async () => {
```

Comment:
"Discarded sectors should be aborted if still pending", I don't really understand what this means 😅

Reply:
Try a test name conventioned closer to, "[methodName][state][expectedBehaviour]"

Something like: "Sector downloads of downloading discarded sectors should abort"